shared/js/background/pixel.es6.js

```diff
@@ -90,7 +90,8 @@ function concatParams (args) {
         if (typeof arg === 'object') {
             objParamString += Object.keys(arg).reduce((params, key) => {
                 const val = arg[key]
-                if (val) return `${params}&${key}=${val}`
+                // val can be zero
+                if (typeof val !== 'undefined') return `${params}&${key}=${val}`
```

Comment:
This is going to allow `val` to be null or an empty string as well.. Do we care?